src/cascadia/QueryExtension/ExtensionPalette.xaml

```diff
@@ -352,18 +352,27 @@
                             </Style>
                         </ListView.ItemContainerStyle>
                     </ListView>
-                    <Button x:Name="ClearMessagesButton"
-                            x:Uid="ClearMessagesButton"
-                            Grid.Row="0"
-                            Grid.Column="2"
-                            Margin="0,0,8,0"
-                            VerticalAlignment="Top"
-                            Click="_clearAndInitializeMessages">
-                        <Button.Content>
-                            <FontIcon FontSize="16"
-                                      Glyph="&#xE81C;" />
-                        </Button.Content>
-                    </Button>
+                    <StackPanel Grid.Row="0"
+                                Grid.Column="2"
+                                Margin="0,0,8,0"
+                                VerticalAlignment="Top"
+                                Orientation="Horizontal"
+                                Spacing="8">
+                        <Button x:Uid="ClearMessagesButton"
+                                Click="_clearAndInitializeMessages">
+                            <Button.Content>
+                                <FontIcon FontSize="16"
+                                          Glyph="&#xE81C;" />
+                            </Button.Content>
+                        </Button>
+                        <Button x:Uid="ExportMessagesButton"
+                                Click="_exportMessagesToClipboard">
+                            <Button.Content>
+                                <FontIcon FontSize="16"
+                                          Glyph="&#xEDE1;" />

```

Comment:
Since we're copying to the clipboard, might be better to use the clipboard glyph instead
![clipboard glyph entry](https://github.com/user-attachments/assets/0ba0e5ac-82d4-4679-8229-e0842d167245)

Reply:
Resolving since the button has been reworked to export to file